src/annotator/sidebar.tsx

```diff
@@ -203,7 +203,14 @@ export class Sidebar implements Destroyable {
 
           // Use a grey background, with lower opacity with the sidebar is
           // collapsed, so the page content behind it can be read.
-          'bg-grey-2 sidebar-collapsed:bg-black/[.08]'
+          'bg-grey-2 sidebar-collapsed:bg-black/[.08]',
+
+          // When the sidebar is closed, allow pointer events to go through
+          // this container to page elements (eg. scroll bar thumbs) which
+          // are behind it. When open and not in side-by-side mode, we enable
+          // pointer events as otherwise clicks in-between buckets could cause
+          // the sidebar to close, which is annoying.
+          'sidebar-collapsed:pointer-events-none'
```

Comment:
TBD for next week: Also enable `pointer-events: none` when the sidebar is open and we are in side-by-side mode. Or just enable it always and accept the annoyance of accidentally closing the sidebar if clicking on the sidebar background.

Reply:
I've thought more about this and decided to change the approach. Since it is the guest which knows whether it is going to request the sidebar be closed or not, it is easer to make a different decision depending on the sidebar's current state in the code which sends that request, instead of here. So what I'm planning to do instead is:

1. Always set `pointer-events: none` on the sidebar container. This will allows events in transparent areas of the sidebar to always go through to the document.
2. In the guest, check the coordinates of mouse or touch events that would cause the guest to close the sidebar, and ignore them if they are within the sidebar's bounds (which the guest already knows about)